tests/test_interface.rb

```diff
@@ -682,9 +712,8 @@ def test_negotiate_auto_ethernet
     # Some platforms/interfaces/versions do not support negotiation changes
     begin
       interface.negotiate_auto = false
-    rescue => e
-      skip('Skip test: Interface type does not allow config change') if
-        e.message[/requested config change not allowed/]
+    rescue RuntimeError => e
+      speed_change_disallowed(e.message)
```

Comment:
@chrisvanheuveln This seems to have been an incorrect fix - speed_change_disallowed doesn't match the `/requested config change not allowed/` pattern so this test case is now failing for me.